WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1284,6 +1290,31 @@ public void onDismiss(DialogInterface dialog) {
         return view;
     }
 
+    private boolean isValidIpOrRange(String entry) {
+        if (TextUtils.isEmpty(entry)) {
+            return false;
+        }
+
+        // remove whitespace
+        entry = entry.trim();
+        entry = entry.replaceAll(" ", "");
+        String[] ipStrings = entry.split("-");
+
+        // should not be more than 2 IP address strings (IP range) in an entry
```

Comment:
Could you write which rules you followed here? I have checked a bit and this is a different check than I found, so I want to make sure it's valid before giving the :shipit:. We should even consider adding the whole rule as a comment.